oe_webtools_analytics/src/EventSubscriber/AnalyticsEventSubscriber.php

```diff
@@ -46,13 +46,11 @@ class AnalyticsEventSubscriber implements EventSubscriberInterface {
    */
   public function __construct(ConfigFactoryInterface $configFactory, RequestStack $requestStack) {
     // Get id from settings.php!
-    if ($configFactory->get('oe_webtools.analytics')) {
-      $this->config = $configFactory->get('oe_webtools.analytics');
-      $this->requestStack = $requestStack;
-    }
-    else {
+    if (!$configFactory->get('oe_webtools.analytics')) {
       throw new \InvalidArgumentException(t('The "oe_webtools.analytics" settings is missing from settings.php', [], ['context' => 'oe_webtools']));
     }
```

Comment:
Usually we had an empty line after the `if` for better readability.

Reply:
Sorry but I don't agree and the Drupal CS does not mention it!

Reply:
I know but I don't agree with the "Drupal CS does not mention it".

It's true that nothing "regulates" the usage of empty lines and also do not consider Drupal CS to be the holy grail, it has many inconsistencies.

Usually I add spaces to clearly see the different condition blocks. But I'm not the only one.
Famous libraries are doing it "by default" (see Geocoder: https://github.com/geocoder-php/Geocoder/blob/master/src/Provider/GoogleMaps/GoogleMaps.php#L105) or PHPunit (https://github.com/sebastianbergmann/phpunit/blob/master/src/Util/ErrorHandler.php) (there are many examples in the code)

They are also adding an empty line before the `return` statement.

Now, of course this is not a blocker at all, it's up to you to decide to follow this or not.

Reply:
I would suggest we bring this discussion to the retro and we set up the CS we are going to follow.

As you mention there are several libraries using it, that's true, but I can say that there are several core Drupal modules not using it (for example node module), so this shows that we need to have a team agreement for this kind of matters!
cc: @ademarco @voidtek @imanoleguskiza @sandervd

Reply:
Mmhm.

As it's not a blocker, I don't think we should spend time "once again" in retro and/or any meeting for dealing with this.
And as it's not in a CS it's up to you to decide to do it or not.

We must think about being flexible and we of course can agree to disagree without doing meetings...